src/jobsautoreport/slack.py

```diff
@@ -1,9 +1,9 @@
 import logging
-from typing import Any
+from typing import Any, Callable, Union
 
+import plotly.graph_objects as go  # type: ignore
```

Comment:
```suggestion
import plotly.graph_objects as graph_objects  # type: ignore
```

``go`` means nothing in the context that you use it. ``graph_objects`` is much clearer 
same goes for almost any abbreviation, unless it's a very common one